modules/aws/sonar-upgrader/python_upgrader/tests/test_main.py

```diff
@@ -414,7 +418,7 @@ def test_main_hadr_set_skip_node_after_hadr_postflight_failure_stop_on_failure_f
 
 @pytest.mark.xfail(raises=UpgradeException)
 def test_main_raise_exception_on_overall_status_failed(
-        args, upgrade_status_service_mock, test_connection_mock, run_remote_script_mock, mocker):
+        args, upgrade_status_service_mock, test_connection_mock, run_remote_script_mock, collect_facts_mock, mocker):
     # given
     setup_custom_args(args, [{"main": gw1}], [], True, True, True, True, True)
     run_remote_script_mock.side_effect = create_mocked_run_remote_script_side_effects(python_location_error_hosts=["host1"])
```

Comment:
the first param python_location_error_hosts in create_mocked_run_remote_script_side_effects function is irrelevant since there is no "get_python_location.sh" anymore. Please remove it so it won't be confusing. Make sure to adjust the places that used this param